src/pretix/presale/templates/pretixpresale/event/fragment_cart.html

```diff
@@ -94,6 +94,21 @@
                         </dd>
                     </div>
                 {% endif %}
+                {% if event.settings.show_checkin_number_user and line.checkin_count %}
```

Comment:
If only the ticket-usage is shown, the surounding `<dl>` is not output. I will push a fix for this.